pkg/apis/eventing/v1alpha1/subscribable_channelable_validation_test.go

```diff
@@ -37,7 +37,7 @@ var validationTests = []struct {
 			Kind:       "Strait",
 		},
 		want: &apis.FieldError{
-			Message: "invalid value \"Strait\"",
+			Message: "invalid value: Strait",
```

Comment:
I think we want these to still be quoted.

Reply:
This came along with picking up the latest knative/pkg and was not relevant to my change.

https://github.com/knative/pkg/commit/f55c11c3ce2dbeb895307455e56e5dba9573243e

Reply:
@n3wscott Are you okay with this?